risc0/zkvm/src/host/server/prove/dev_mode.rs

```diff
@@ -74,7 +74,9 @@ impl ProverServer for DevModeProver {
         let mut keccak_assumptions = HashMap::new();
 
         for proof_request in session.pending_keccaks.iter() {
-            let claim = compute_keccak_digest(&proof_request.input);
+            // Note: safe to cast as slice here, given alignment of KeccakState (8 bytes) is greater
+            // than alignment of u8 slice.
```

Comment:
because a `u8` slice has no alignment requirement, its always going to be okay to cast some memory to it.